Header header
string hover_point

float64 north_cmd
float64 east_cmd
float64 north_vel_cmd
float64 east_vel_cmd
float64 yaw_cmd

float64 alt_override

float64 north_err
float64 east_err
float64 north_vel_err
float64 east_vel_err
float64 yaw_err